airmozilla/manage/templates/manage/survey_edit_questions.html

```diff
@@ -0,0 +1,82 @@
+{% extends 'manage/surveys.html' %}
+
+{% block manage_title %}
+    Questions for "{{survey.name}}"
+{% endblock %}
+
+{% block site_css %}
+  {{ super() }}
+  {% compress css %}
+  <link href="{{ static('select2/select2.css') }}" rel="stylesheet" type="text/css">
+  <link href="{{ static('select2/select2-bootstrap.css') }}" rel="stylesheet" type="text/css">
+  <link href="{{ static('manage/css/survey.css') }}" rel="stylesheet" type="text/css">
+  {% endcompress %}
+{% endblock %}
+
+{% block site_js %}
+  {{ super() }}
+  <script src="{{ static('select2/select2.min.js') }}"></script>
+  {% compress js %}
+    <script src="{{ static('manage/js/survey-edit.js') }}"></script>
+  {% endcompress %}
+{% endblock %}
+
+{% block content_class %}col-md-6{% endblock %}
+
+{%block manage_content %}
+
+      <table class="table table-striped table-bordered">
+      <thead>
+        <tr>
+          <th>JSON</th>
+          <th>Order</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+      {% for question in questions %}
+      <tr>
+          <td class="question">
+            <textarea name="question">{{ json_print(question.question, indent=2) }}</textarea>
+          </td>
+          <td class="ordering">
+            <form method="post" action="{{ url('manage:survey_question_edit', survey.id, question.id) }}">
+              {{ csrf() }}
+              <button class="btn btn-sm ordering-up" name="ordering" value="up"><i class="glyphicon glyphicon-arrow-up"></i></button><br>
+              <button class="btn btn-sm ordering-down" name="ordering" value="down"><i class="glyphicon glyphicon-arrow-down"></i></button>
+            </form>
+          </td>
+          <td>
+            <form method="post" style="display:inline" class="edit"
+                  action="{{ url('manage:survey_question_edit', survey.id, question.id) }}">
+              {{ csrf() }}
+              <button class="btn btn-primary btn-block">
+                <i class="glyphicon glyphicon-floppy-disk"></i>
+                Save
+              </button>
+            </form>
+            <form method="post" class="confirm"
+                  action="{{ url('manage:survey_question_delete', survey.id, question.id) }}">
+              {{ csrf() }}
+              <button class="btn btn-danger btn-block" type="submit">
+                <i class="glyphicon glyphicon-trash"></i>
+                Delete
+              </button>
+            </form>
+          </td>
+        </tr>
+      {% endfor %}
+      </tbody>
+    </table>
+
+    <form method="post" action="{{ url('manage:survey_question_new', survey.id) }}">
+      {{ csrf() }}
+      <button class="btn btn-primary">
+        <i class="glyphicon glyphicon-plus-sign"></i>
+        New question
+      </button>
+    </form>
+
+    <a class="survey-links" href="{{ url('manage:survey_edit', survey.id)}}">Back to editing "{{survey.name}}"</a>
```

Comment:
Same thing here. You'll see other examples of this, [like this](https://github.com/mozilla/airmozilla/blob/master/airmozilla/manage/templates/manage/event_discussion.html#L50-L52)